pkgs/tools/misc/pastebinit/default.nix

```diff
@@ -47,6 +45,6 @@ stdenv.mkDerivation rec {
     description = "A software that lets you send anything you want directly to a pastebin from the command line";
     maintainers = with maintainers; [ raboof ];
     license = licenses.gpl2;
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ lib.platforms.darwin;
```

Comment:
(I can't confirm darwin compatibility but will take your word for it ;) )

Reply:
It runs on my Mac without any deprecation warnings :)